core/src/com/mygdx/game/config/Config.java

```diff
@@ -2,7 +2,7 @@
 
 import lombok.NonNull;
 
-public interface EntityConfig {
```

Comment:
nie jestem pewien, Config jest mniej konkretne, wolałbym EntityConfig, i zmiana inta na long moim zdaniem nie potrzebna, tak samo we wszystkich implementacjach z tym longiem

Reply:
int/long nie ma znaczenia za bardzo, ale jest "bezpieczniejsze" pod względem tego, że łatwiej tak wprowadzać nowe configi, tylko trzeba strzelić jakiś zakres gdzieśtam zasięgu kilkudziesięciu tysięcy do przodu i nastawić się, że następne 10 tysięcy to jest ten "typ"